catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -27,44 +31,47 @@ function getFirstPhysicalLocation(item) {
   return item.locations?.find(location => location.type === 'PhysicalLocation');
 }
 
-function createBodyRow(
+function createDescriptionList(
   item: PhysicalItem,
   encoreLink: string | undefined
-): (string | ReactElement)[] | undefined {
+): DescriptionListProps | undefined {
   const physicalLocation = getFirstPhysicalLocation(item);
   const isRequestableOnline =
     physicalLocation?.accessConditions?.[0]?.method?.id === 'online-request';
   const accessMethodLabel =
-    physicalLocation?.accessConditions?.[0]?.method?.label || ' ';
+    physicalLocation?.accessConditions?.[0]?.method?.label || '';
   const accessMethod =
     isRequestableOnline && encoreLink ? (
       <ButtonInlineLink text={accessMethodLabel} link={encoreLink} />
     ) : (
       <>{accessMethodLabel}</>
     );
   const accessStatus =
-    physicalLocation?.accessConditions?.[0]?.status?.label || ' ';
-  const accessTerms = physicalLocation?.accessConditions[0]?.terms || ' ';
+    physicalLocation?.accessConditions?.[0]?.status?.label || '';
+  const accessTerms = physicalLocation?.accessConditions[0]?.terms || '';
   const locationId = physicalLocation?.locationType.id;
   const locationLabel = physicalLocation && getLocationLabel(physicalLocation);
   const locationShelfmark =
     physicalLocation && getLocationShelfmark(physicalLocation);
   const shelfmark = `${locationLabel ?? ''} ${locationShelfmark ?? ''}`;
 
   if (locationId !== 'on-order') {
```

Comment:
I think we should probably get rid of this and the related code from l139 now we're moving in a new direction with the layout.

Reply:
it was when we didn't want stuff on order to be part of the table: https://app.zeplin.io/project/5aab926b4b6efde01259e959/screen/6093a75840e0eb31b669c2f4